### PR TITLE
Release Notes/Known Issues in CNV 1.4

### PR DESCRIPTION
**Pull request by ousleyp**
This PR will include all Known Issues to be included in the CNV 1.4 Release Notes.

I'll request peer review before the release.

### CODE REVIEW

File: cnv_release_notes/cnv_release_notes.adoc [hunk 1]
```diff
@@ -11,6 +11,17 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
+==== Updated KubeVirt API
```
Comment (davidvossel) [inline cnv_release_notes/cnv_release_notes.adoc]: :+1:  this looks accurate for the kubevirt api part.

File: cnv_release_notes/cnv_release_notes.adoc [hunk 2]
```diff
@@ -11,6 +11,17 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+** A new version of the KubeVirt API is included in {ProductName} 1.4. Several 
+important changes are reflected in the latest configuration file templates.
+*** The `apiVersion` has been updated from `kubevirt.io/v1alpha2` to 
+`kubevirt.io/v1alpha3`.
+*** The `volumeName` attribute no longer exists. Ensure that each disk name 
+matches the corresponding volume name in all configuration files.
+*** All instances of `registryDisk` have been updated to read `containerDisk`.
```
Comment (fabiand) [inline cnv_release_notes/cnv_release_notes.adoc]: + "Ensure that all instances of registryDisk get update to containerDisk"

My pont, a user needs to update this in their definitions.

File: cnv_release_notes/cnv_release_notes.adoc [hunk 3]
```diff
@@ -30,46 +41,32 @@ create new virtual machines.
...
-** This will be fixed with the introduction of operators in CNV 1.4.
+
+[[RN_known_issues]]
+== Known issues
+
+* CPU Manager, a feature that provides CPU pinning in {product-title}, is 
+currently disabled in {ProductName} due to performance regressions. CPU Manager 
+does not consider the physical CPU topology, resulting in sub-optimal pinning 
+when hyper-threading is enabled. 
+https://bugzilla.redhat.com/show_bug.cgi?id=1667854[(BZ#1667854)]
```
Comment (fabiand) [inline cnv_release_notes/cnv_release_notes.adoc]: lgtm

**Review by fabiand — COMMENTED** [review]
Just one nit

File: cnv_release_notes/cnv_release_notes.adoc (hunk 2 above)
Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: @fabiand Upgrading from 1.x->1.4 is not supported, correct? Would a user still have files with "registryDisk" to manually update?

**Comment by fabiand** [comment]
Correct. There is no update supported.

**Review by dankenigsberg — APPROVED** [review]

File: cnv_release_notes/cnv_release_notes.adoc [hunk 4]
```diff
@@ -11,65 +11,49 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+==== Updated KubeVirt API
+* A new version of the KubeVirt API is included in {ProductName} 1.4. Several 
+important changes are reflected in the latest configuration file templates.
+** The `apiVersion` has been updated from `kubevirt.io/v1alpha2` to 
+`kubevirt.io/v1alpha3`.
+** The `volumeName` attribute no longer exists. Ensure that each disk name 
+matches the corresponding volume name in all configuration files.
+** All instances of `registryDisk` must be updated to `containerDisk` in
+configuration files.
+
```
Comment (rthallisey) [inline cnv_release_notes/cnv_release_notes.adoc]:
```
=== Introducing Operators
** The new install of KubeVirt, Containerized Data Imported, and web-ui is powered by Operators.
```

Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: Thanks Ryan, check my latest update. I reworded it slightly:

=== Introducing Operators
* Operators power the installation of KubeVirt, Containerized Data Importer, and
the *web-ui* in {ProductName} 1.4.

**Comment by ousleyp** [comment]
Preview build: http://file.bos.redhat.com/pousley/022019/knownissues/cnv_release_notes/cnv_release_notes.html

File: cnv_release_notes/cnv_release_notes.adoc [hunk 5]
```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+* If you create a new DataVolume while a PVC already exists with the same name,
+the DataVolume enters an unrecoverable error state. If this DataVolume is
+associated with a virtual machine, or if it was created with the
+`dataVolumeTemplates` section of a VM configuration file, then the VM will fail
+to start. In these cases, the underlying DataVolume error will not be propagated
+to the VM. https://bugzilla.redhat.com/show_bug.cgi?id=1669163[(BZ#1669163)]
+
+* If a CDI import into a PVC fails, a request to delete the PVC might not work
+immediately. Instead, the importer Pod gets stuck in a `CrashLoopBackOff` state,
```
Comment (aburdenthehand) [inline cnv_release_notes/cnv_release_notes.adoc]: s/importer Pod/importer pod/

File: cnv_release_notes/cnv_release_notes.adoc [hunk 6]
```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+* The `runc` package version `runc-1.0.0-54` contained a bug that caused the
+virt-launcher to crash if FIPS was disabled. The version of `runc` containing
+the fix is now shipped with Red Hat Enterprise Linux 7 Extras.
+https://bugzilla.redhat.com/show_bug.cgi?id=1650512[(BZ1650512)]
+
+// Below issues are expected to be resolved, but may need double-checking
+
+* In the Create VM Wizard, using the *PXE* source option with the *Start VM on creation*
+option resulted in the boot order not changing after stopping and
+starting the VM. This issue has been resolved.
```
Comment (aburdenthehand) [inline cnv_release_notes/cnv_release_notes.adoc]: s/VM/virtual machine/

File: cnv_release_notes/cnv_release_notes.adoc [hunk 7]
```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+====
+The command `$ ls -sh <image_file>` reports the actual size. For uncompressed raw
+images, the command `$ ls -lh <image_file>` reports the virtual size. For QCOW2
+images, use `$ qemu-img info <image_file>` to get both values.
+====
+
+* If you create a new DataVolume while a PVC already exists with the same name,
+the DataVolume enters an unrecoverable error state. If this DataVolume is
+associated with a virtual machine, or if it was created with the
+`dataVolumeTemplates` section of a VM configuration file, then the VM will fail
```
Comment (aburdenthehand) [inline cnv_release_notes/cnv_release_notes.adoc]: as above: s/VM/virtual machine/g

File: cnv_release_notes/cnv_release_notes.adoc [hunk 8]
```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
-==== Virtual machine control from the web console
-** The updated {ProductName} web console now allows users to create virtual 
-machines, access virtual machines with either a serial or VNC connection, and 
-perform basic power management tasks (start|stop) on virtual machines. 
+=== Introducing Operators
+* Operators power the installation of KubeVirt, Containerized Data Importer, and
+the *web-ui* in {ProductName} 1.4.
```
Comment (aburdenthehand) [inline cnv_release_notes/cnv_release_notes.adoc]: Is *web-ui* meant to be marked up? 
Ignore me if it is.

File: cnv_release_notes/cnv_release_notes.adoc [hunk 9]
```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
...
+== Resolved issues
+
+* The `runc` package version `runc-1.0.0-54` contained a bug that caused the
+virt-launcher to crash if FIPS was disabled. The version of `runc` containing
+the fix is now shipped with Red Hat Enterprise Linux 7 Extras.
+https://bugzilla.redhat.com/show_bug.cgi?id=1650512[(BZ1650512)]
+
+// Below issues are expected to be resolved, but may need double-checking
+
+* In the Create VM Wizard, using the *PXE* source option with the *Start VM on creation*
```
Comment (aburdenthehand) [inline cnv_release_notes/cnv_release_notes.adoc]: s/Create VM/*Create Virtual Machine*/
Also on this line: the checkbox is *Start virtual machine on creation*

File: cnv_release_notes/cnv_release_notes.adoc (hunk 5 above)
Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: Ahh, I forgot that it was "pod" in 3.11. I had recently referred to the 4.0 glossary where it's listed as "Pod." Thanks, @aburdenthehand :)

File: cnv_release_notes/cnv_release_notes.adoc (hunk 8 above)
Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: I don't think it should be. Removed the asterisks.

**Pull request merged**